web/docs/GameTranslation.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import Alert from '@material-ui/lab/Alert';
+
+<Meta title="Documentation/Game Translation" />
+
+# Game translation
+
+We recommend every game to be translate to all languages available. The following tutorial teaches how to add translations to a game.
```

Comment:
I like this idea of keeping it focused to a singe game :). We can create another one more broadly on adding a new language